.github/workflows/dhis2-verify-commits.yml

```diff
@@ -0,0 +1,44 @@
+name: 'dhis2: verify (commits)'
+
+on:
+    pull_request:
+        types: ['opened', 'edited', 'reopened', 'synchronize']
+
+jobs:
+    lint-pr-title:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: 18.x
```

Comment:
Should we start using 20.x?
GHA transitioned all actions to Node v20 already in spring 2024.